templates/Default/index.html.twig

```diff
@@ -5,7 +5,7 @@
     <h3>Links to:</h3>
     <ul>
         <li><a href="{{  path("pelagos_app_ui_account_default") }}">Request Account</a></li>
-        <li><a href="{{  path("pelagos_app_ui_account_passwordreset") }}">Password Reset</a></li>
+        <li><a href="{{  path("pelagos_app_ui_account_password") }}">Change/Reset password</a></li>
```

Comment:
Text change is fine, but the route name change is not necessary and it needs to be changed in the Controllers as well, if we want to make this change

Reply:
Never mind, I thought this route dint exist